x-pack/plugins/observability/server/services/slo/update_slo.ts

```diff
@@ -41,7 +42,7 @@ export class UpdateSLO {
 
   private updateSLO(originalSlo: SLO, params: UpdateSLOParams) {
     let hasBreakingChange = false;
-    const updatedSlo: SLO = Object.assign({}, originalSlo, params, { updated_at: new Date() });
+    const updatedSlo: SLO = merge({}, originalSlo, params, { updatedAt: new Date() });
```

Comment:
💬  lodash's merge is better than a simple Object.assign as it merge recursively the objects